### DISCUSSION TITLE
Rename master to main?

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4186] [Ideas]
Given the migration of the community and `git` tooling away from `master` to `main`, are there any opinions on whether `iris` should adopt this rebrand?

See [here](https://github.com/github/renaming) for further details.

What do you think @SciTools/iris-devs ? :thinking: 

P.S. Also see this similar [discussion](https://github.com/SciTools/nc-time-axis/discussions/65) on `nc-time-axis`.

**Comment by jamesp** [comment]
Small change, easy to make.  👍

**Comment by rcomer** [comment]
This might be already in hand, but I noticed the [latest docs](https://scitools-iris.readthedocs.io/en/latest/) are stuck on commit https://github.com/SciTools/iris/commit/9268ca9c70bd160c35c5099025d632ae33faf858, which I think was the last one on `master`.

**Reply by rcomer** [reply]
Looks like that's now sorted!

**Comment by rcomer** [comment]
Within my local clone, it seems I can still "see" and even rebase against `upstream/master`.  Is there a way I can get rid of that?

**Reply by rcomer** [reply]
I have answered my own question:

```
git fetch -p upstream
```

p is for prune

**Comment by pp-mo** [comment]
# We have done this : replaced "master" with "main" as default branch

I will now use this as a trial of an idea :
Since you cannot 'close' a discussion, only _delete_ it, 
how would we retain the discussion while 'closing' it ?

So, I suggest
**To archive a discussion, convert to Issue + Close**

**Reply by pp-mo** [reply]
Oh, it doesn't work.
"Convert to Issue" only copies the head -- it doesn't take the comments with it, or duplicate them

**Comment by pp-mo** [comment]
> Since you cannot 'close' a discussion, only delete it,
how would we retain the discussion while 'closing' it ?

Another idea:
re-categorise as "Archived"
